drivers/ovsSwitch.go

```diff
@@ -326,9 +327,10 @@ func (sw *OvsSwitch) CreatePort(intfName string, cfgEp *mastercfg.CfgEndpointSta
 }
 
 // UpdatePort updates an OVS port without creating it
-func (sw *OvsSwitch) UpdatePort(intfName string, cfgEp *mastercfg.CfgEndpointState, pktTag int) error {
+func (sw *OvsSwitch) UpdatePort(intfName string, cfgEp *mastercfg.CfgEndpointState, pktTag int, nwType string) error {
+
```

Comment:
This API is exposed to (and invoked by) ovsdriver which is aware only about intfName and not OVS Port name. So to be consistent with CreatePort, retained this API to take intfName and passing a skipVethPair flag instead of nwType to this API. Hope this sounds fine.